r/tests/testthat/test-metadata.R

```diff
@@ -214,15 +214,27 @@ test_that("metadata drops readr's problems attribute", {
   expect_null(attr(as.data.frame(tab), "problems"))
 })
 
-test_that("metadata of list elements (ARROW-10386)", {
+test_that("Row-level metadata (does not by default) roundtrip", {
+  # First tracked at ARROW-10386, though it was later determined that row-level
+  # metadata should be handled separately ARROW-14020, ARROW-XXX
```

Comment:
I'm a little afraid to ask: what is ARROW-XXX?

Reply:
HA, it was a placeholder for putting in ticket numbers for what we _hope_ to do in the future to be able to save this without taking this approach.